### DISCUSSION TITLE
navidrome.service not starting autmaticallyt after reboot

### DISCUSSION THREAD

**Discussion by vinny2474** [navidrome/navidrome#2839] [Q&A]
Hello, new to Navidrome,I have tested it few hours, then had to reboot my Ubuntu desktop PC. After reboot, I had to manually restart navidrome.service : is it supposed to restart automatically after reboot ?
Tks,

**Comment by metalheim** [comment]
Depends 100% on how you installed navidrome

**Comment by vinny2474** [comment]
I have followed the installation instructions for Linux on Navidrome website. I have made a wrong install using bad packet, then reinstall with amd64. I have just seen that I forgot to do this command : sudo systemctl enable navidrome.service
I try it but surely that is why the service didn't restarted automatically after reboot ;-).
Thanks for the hint
